src/qililab/qprogram/experiment_executor.py

```diff
@@ -104,9 +104,10 @@ class ExperimentExecutor:
         - The results will be saved in a timestamped directory within the `base_data_path`.
     """
 
-    def __init__(self, platform: "Platform", experiment: Experiment):
+    def __init__(self, platform: "Platform", experiment: Experiment, live_plot: bool = False):
```

Comment:
Do we need this flag? should not be by default the experimental option? Have we checked with the experimentalists on the behaviour of this future when it comes to turning it on and off, if needed to be turn on and off like this flag does?

Reply:
That is a very good point, indeed the desired behaviour is to have liveplot, I don't think it will hurt to have an option to disable it but definitely the default option should be True

Reply:
Yes, let's confirm with Oscar.

Reply:
I think we should always allow the users to not use live plotting, and just run an experiment.